tests/e2e/vsan_test.go

```diff
@@ -36,6 +38,7 @@ import (
 )
 
 const (
+	vsanTest       = "vsan_test"
 	vsanVolumeTest = "vsan_vol"
 	vsanPolicyFlag = "vsan-policy-name"
```

Comment:
correct! 

By taking `vsanOpts := " -o " + vsanPolicyFlag + "=" + policyName` as an example; it is going to be consumed by multiple files in the future. 

Anyway this is a trivial comment, my only intention is to avoid future refactoring.